sp800-63b/sec5_authenticators.md

```diff
@@ -119,9 +119,9 @@ If the out-of-band authenticator sends an approval message over the secondary co
 
 ##### 5.1.3.2. Out-of-Band Verifiers
 
-Due to the risk that SMS messages or voice calls may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out-of-band verification is to be made using the public switched telephone network (PSTN), the verifier SHALL verify that the pre-registered telephone number being used is not associated with a VoIP (or other software-based) service. It then sends the SMS or voice message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.
+If out-of-band verification is to be made using the public switched telephone network (PSTN), the verifier SHALL verify that the pre-registered telephone number being used is associated to a physical device. The verifier then sends the authentication secret to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change. Verifiers SHALL use known and verifiable routes to deliver the secret. Verifiers SHOULD be aware of indicators such as device swap, SIM change, number porting, or other abnormal behavior *before* using the OOB channel to deliver the authentication secret.
```

Comment:
The second sentence assumes a particular out-of-band flow, while below (2-3 paragraphs later), we describe three possible flows (added in response to a preview draft comment). At least the second of these is vulnerable to spoofing of the source of the text message from the claimant to the verifier.